tv_show/src/components/shared/shows/ShowsList/ShowsList.test.tsx

```diff
@@ -24,6 +29,16 @@ describe('ShowsList', () => {
 	it('should render all provided shows', () => {
 		render(<ShowsList shows={mockShows} />);
 
-		expect(screen.getAllByRole('link').length).toEqual(mockShows.length);
+		expect(ShowCard).toHaveBeenCalledTimes(2);
+	});
+
+	it('should render ShowCard', () => {
+		render(<ShowsList shows={mockShows} />);
+
+		{
+			mockShows.map((show) => {
+				expect(ShowCard).toHaveBeenCalledWith({ show }, expect.anything());
```

Comment:
Točno ali ima i opcija:
`toHaveBeenNthCalledWith` (da smo sigurno za redoslijed pozivanja)